charts/mgmt-agent/templates/env-configmap.yaml

```diff
@@ -18,3 +18,8 @@ data:
 
   # Modifiable properties
   # Add new or change the below existing properties as required
+
+  # Openjdk on RHEL with FIPS requires the default secruity property to be disabled, so that 
```

Comment:
Openjdk on RHEL with FIPS requires the default secruity property to be disabled, so that Management Agent can use its own bcfips security provider